From bf28ff1f422407f10ecc283f03ce8c5ee57f1812 Mon Sep 17 00:00:00 2001
From: Brad King <brad.king@kitware.com>
Date: Wed, 5 Mar 2014 11:57:00 -0500
Subject: [PATCH] FindPythonLibs: Do not try to find the interpreter (#13794)

The parent commit taught FindPythonLibs to try to find PythonInterp
unconditionally.  Some projects may want the libraries of a specific
version even when the corresponding interpreter is not available.  Drop
the internal use of FindPythonInterp and just use the versions from it
if it happens to have been found by the project first.  That will allow
projects to get a consistent version when they want both but not
otherwise force them to find the interpreter.
---
 Modules/FindPythonLibs.cmake |    1 -
 1 file changed, 1 deletion(-)

diff --git a/Modules/FindPythonLibs.cmake b/Modules/FindPythonLibs.cmake
index 27d9e45..656ed7b 100644
--- a/Modules/FindPythonLibs.cmake
+++ b/Modules/FindPythonLibs.cmake
@@ -83,7 +83,6 @@ endif()
 # If FindPythonInterp has already found the major and minor version,
 # insert that version between the user supplied versions and the stock
 # version list.
-find_package(PythonInterp QUIET)
 set(_Python_VERSIONS ${Python_ADDITIONAL_VERSIONS})
 if(DEFINED PYTHON_VERSION_MAJOR AND DEFINED PYTHON_VERSION_MINOR)
   list(APPEND _Python_VERSIONS ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
-- 
1.7.10.4

